docs/adr/001-file-metadata-mariadb.md

```diff
@@ -0,0 +1,90 @@
+# 📝 Architecture Decision Record (ADR)
+
+## Title  
+Storing File Metadata and Extracted Entities in MariaDB
```

Comment:
The title makes it look like a post hoc justification rather than an ADR. I think the title should be something like _Selecting DB storage for extracted entities_.